hibernate-core/src/main/java/org/hibernate/hql/internal/ast/tree/AbstractSelectExpression.java

```diff
@@ -55,4 +61,40 @@ public void setScalarColumn(int i) throws SemanticException {
 	public int getScalarColumnIndex() {
 		return scalarColumnIndex;
 	}
+
+	protected static String[] extractMutationTexts(Node operand, int count) {
```

Comment:
Do we have reason to use the combination of 'protected' + 'static'? I even googled for this: https://www.google.com/search?q=protected+static+java+method&rlz=1C5CHFA_enCA943CA943&oq=protected+static+java+method&aqs=chrome..69i57j0i22i30l3.4244j1j7&sourceid=chrome&ie=UTF-8